prisma/seeds/networks.ts

```diff
@@ -1,35 +1,40 @@
-import { NETWORK_SLUGS } from '../../constants/index'
+import { NETWORK_SLUGS, NETWORK_IDS } from '../../constants/index'
 
 export const networks = [
   {
+    id: NETWORK_IDS.XEC,
     slug: NETWORK_SLUGS.ecash,
     ticker: 'xec',
     title: 'eCash',
     createdAt: new Date(),
     updatedAt: new Date()
   },
   {
+    id: NETWORK_IDS.BCH,
     slug: NETWORK_SLUGS.bitcoincash,
     ticker: 'bch',
     title: 'Bitcoin Cash',
     createdAt: new Date(),
     updatedAt: new Date()
   },
   {
+    id: 3,
```

Comment:
Any point in making these UUIDs or better to just leave them?

Reply:
I don't see why, since those `Network` objects are about the project structure, not about user data. They are only inserted /deleted through migrations (actually only inserted)